src/commands/react_native_xcode.rs

```diff
@@ -77,6 +77,15 @@ pub fn make_app<'a, 'b: 'a>(app: App<'a, 'b>) -> App<'a, 'b> {
                      to use.  By default the bundled build script is used.",
                 ),
         )
+        .arg(
+            Arg::with_name("dist")
+                .long("dist")
+                .value_name("DISTRIBUTION")
+                .required(true)
```

Comment:
This will break the current CLI as it requires the `dist` argument to be passed. Can you remove this line, please?